packages/webapp/src/Routes.jsx

```diff
@@ -316,6 +316,114 @@ const UnknownRecord = React.lazy(() =>
   import('./containers/ErrorHandler/UnknownRecord/UnknownRecord'),
 );
 
+const FinancesRoutes = () => (
+  <Switch>
+    <Route path="/finances/transactions" exact component={Finances} />
+    <Route path="/finances/actual_revenue" exact component={ActualRevenue} />
+    <Route
+      path="/finances/estimated_revenue/plan/:management_plan_id"
+      exact
+      component={UpdateEstimatedCropRevenue}
+    />
+    <Route path="/finances/revenue_types" exact component={RevenueTypes} />
+    <Route path="/finances/add_sale" exact component={AddSale} />
```

Comment:
I think we didn't get around to changing these url names during the last update. But wondering if we should be moving all references to "revenue" from "sale" here since we are improving our routes structure.

Reply:
For sure, can do!

Reply:
I only changed this one which is customer facing, because changing the component names and params inside the components will make this PR quite a lot bigger and I want to make it easier to review. I'll open up a follow up PR to clean them up

Reply:
Sounds good!